core/FrontController.php

```diff
@@ -133,6 +148,15 @@ public function dispatch($module = null, $action = null, $parameters = null)
              * @param \Piwik\NoAccessException $exception The exception that was caught.
              */
             Piwik::postEvent('User.isNotAuthorized', array($exception), $pending = true);
+        } catch (\Twig_Error_Runtime $e) {
+            echo $this->generateSafeModeOutputFromException($e);
```

Comment:
Here we're catching the case of #10738 `An exception has been thrown during the rendering of a template (".form-description is undefined: failed at `.form-description;` line: 6182") in "@CoreUpdater/runUpdaterAndExit_welcome.twig" at line 8.`